apps/landing/components/Terms/Content.tsx

```diff
@@ -0,0 +1,394 @@
+// TODO: convert this component to server component once the Typography task is done
+"use client";
+
+import React from "react";
+import { useFormatMessage } from "@/hooks/intl";
+import { Typography } from "@litespace/ui/Typography";
+
+import Facebook from "@litespace/assets/Facebook";
+import Youtube from "@litespace/assets/Youtube";
+import Insta from "@litespace/assets/Instagram";
+import Linkedin from "@litespace/assets/Linkedin";
+import Whatsapp from "@litespace/assets/Whatsapp";
+
+const Terms: React.FC = () => {
+  const intl = useFormatMessage();
+  return (
+    <div className="flex flex-col py-14 sm:py-20 md:py-20 px-4 md:px-8 lg:px-[108px] gap-14 sm:gap-20">
```

Comment:
The content should has a max width of `3xl` - like what we are doing in the web workspace - and centered. 

```suggestion
    <div className="flex flex-col py-14 sm:py-20 md:py-20 px-4 md:px-8 lg:px-[108px] gap-14 sm:gap-20 max-w-screen-3xl mx-auto w-full">
```